sql/core/src/test/scala/org/apache/spark/sql/execution/SQLViewSuite.scala

```diff
@@ -812,20 +812,6 @@ abstract class SQLViewSuite extends QueryTest with SQLTestUtils {
     }
   }
 
-  test("creating local temp view should not affect existing table reference") {
```

Comment:
why do we remove this test?

Reply:
This is same to the newly added test case in SQLViewTestSuite.
They both test the scenario:
global temp view refers to a table, create a temporary view with the same name, won't affect the result.

But SQLViewTestSuite will covert all 3 kinds of views.